src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPService.kt

```diff
@@ -196,8 +196,8 @@ class CSIPService(
     currentRequest.reportDetailRequest?.let { reportDetailRequest ->
       involvement = reportDetailRequest.involvementCode?.let { lookupInvolvement(it) }
       concernDescription = reportDetailRequest.concern?.truncateToUtf8Length(4000)
-      knownReasons = reportDetailRequest.knownReasons
-      otherInformation = reportDetailRequest.otherInformation
+      knownReasons = reportDetailRequest.knownReasons?.truncateToUtf8Length(4000)
```

Comment:
CSIP aren't interested in the "see DPS for more details" suffix - so used same truncate method as other csip fields